src/managers/input/InputManager.cpp

```diff
@@ -1207,8 +1207,10 @@ void CInputManager::onKeyboardKey(wlr_keyboard_key_event* e, SKeyboard* pKeyboar
         updateKeyboardsLeds(pKeyboard->keyboard);
     }
 
-    if (m_bExitTriggered)
+    if (m_bExitTriggered) {
+        g_pCompositor->m_bShouldSaveCrash = false;
```

Comment:
wouldn't it be better to put this inside cleanup? this isn't the only place that triggers the cleanup.

Reply:
The point is only when manually exiting, skip the crash report

Reply:
cleanup doesn't get called on crash